### PR TITLE
Refactor panel.get_view_lines

### PR DESCRIPTION
**Pull request by kaste**
For @pykong with ❤️

### CODE REVIEW

File: panel/panel.py [hunk 1]
```diff
@@ -163,30 +162,18 @@ def filter_ok(check_val, key, panel_filter):
...
-    for lineno, line_dict in sorted(view_dict["line_dicts"].items()):
-        items = []
-        for error_type in WARN_ERR:
-            if not filter_ok(error_type, "types", panel_filter):
-                continue
+    return sorted(
+        (item
+         for lineno, line_dict in view_dict["line_dicts"].items()
+         for error_type, items in line_dict.items()
+         for item in items
```
Comment (kaste) [inline panel/panel.py]: This block is nice bc if you change the receiving data structure, you just have to mangle the for..in unpacking here. The rest stays as is.

E.g. if you would consume `raw_errors`, you just delete the first two for..ins and basically just do a `for item in raw_errors[buffer_id]`. And you're done.

Comment (kaste) [reply panel/panel.py]: I'm done for today. 🥃

Comment (pykong) [reply panel/panel.py]: Cheers!

**Comment by braver** [comment]
@pykong do you want to merge this into your branch?

**Comment by pykong** [comment]
As I now started to base the panel logic on kaste's `raw_errors` this PR has become redundant.
We do not even have `panel.get_view_lines` there at all.

@kaste Should we close this PR?

**Pull request closed without merging**